src/main/java/net/coderbot/iris/mixin/MixinMaxFpsCrashFix.java

```diff
@@ -0,0 +1,32 @@
+package net.coderbot.iris.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.Slice;
+
+import net.minecraft.client.Options;
+
+/**
+ * A workaround for when OptiFine has set the maxFps to zero in options.txt
+ * 
+ * Fun.
+ */
+@Mixin(Options.class)
+public abstract class MixinMaxFpsCrashFix {
+	@Redirect(
```

Comment:
Might want to add an `allow = 1` to ensure that this only matches a single injection.

Reply:
Added.